domain/status/state/state.go

```diff
@@ -328,6 +381,61 @@ func (st *State) SetRelationStatus(
 	return nil
 }
 
+// ImportRelationStatus sets the given relation status. It can return the
+// following errors:
+//   - [statuserrors.RelationNotFound] if the relation doesn't exist.
+func (st *State) ImportRelationStatus(
+	ctx context.Context,
+	relationID int,
+	sts status.StatusInfo[status.RelationStatusType],
+) error {
+	db, err := st.DB()
+	if err != nil {
+		return errors.Capture(err)
+	}
+
+	return db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
+		relationUUID, err := st.getRelationUUIDByID(ctx, tx, relationID)
+		if err != nil {
+			return errors.Errorf("getting relation UUID: %w", err)
+		}
+
+		return st.updateRelationStatus(ctx, tx, relationUUID, sts)
```

Comment:
issue: I made the suggested change and relation import no longer writes an initial faux status row, now this call fails silently. Found during QA.

```
repl > select * from relation_status
relation_uuid	relation_status_type_id	suspended_reason	updated_at

repl > 
```

Reply:
I started fixing this here but the code starts getting a lot more complicated than it needs to be. 

In my opinion the correct fix here is to put it back into the relation migration. Then it mirrors the normal realtion lifecycle where it is created in the relation domain and updated in the status domain.